backend/ownerSizeTotalWorker.js

```diff
@@ -0,0 +1,272 @@
+  }))
+}
+
+const lookupUltimateOwner = async (resourceID: string) => {
+  let ownerID = resourceID
+  for (let depth = 128; depth >= 0; depth--) {
+    // Fetch the immediate owner.
+    const newOwnerID = await sbp('chelonia.db/get', `_private_owner_${ownerID}`, { bypassCache: true })
+    // Found the ultimate owner
+    if (!newOwnerID) break
+    if (!depth) {
+      throw new Error('Exceeded max depth looking up owner for ' + resourceID)
+    }
+    ownerID = newOwnerID
+  }
+  return ownerID
+}
+
+/**
+ * Runs once on worker startup or initialization.
+ * It loads all CIDs from the persistent temporary index buckets into the
+ * in-memory `updatedSizeList`.
+ * These CIDs represent items needing a full recalculation (potentially from a
+ * previous unclean shutdown).
+ */
+sbp('okTurtles.eventQueue/queueEvent', readyQueueName, async () => {
+  // Iterate through all 256 possible bucket keys.
+  for (let i = 0; i < 256; i++) {
+    // Fetch the content of the bucket
+    // (bypassCache isn't really needed here as the cache should be empty, but
+    // it's added for clarity.)
+    const data = await sbp('chelonia.db/get', `_private_pendingIdx_ownerTotalSize_${i}`, { bypassCache: true })
+    if (data) {
+      // Split the string and add mark each CID as requiring full recalculation
+      data.split('\x00').forEach((cid) => {
+        updatedSizeList.add(cid)
+      })
+    }
+  }
+
+  console.log(`Loaded ${updatedSizeList.size} CIDs for full recalculation.`)
+  if (updatedSizeList.size) {
+    sbp('backend/server/computeSizeTask')
+  }
+
+  // Schedule the recurring delta computation task
+  setTimeout(sbp, TASK_TIME_INTERVAL, 'backend/server/computeSizeTaskDeltas')
+})
+
+sbp('sbp/selectors/register', {
+  /**
+   * Selector: 'worker/updateSizeSideEffects'
+   * Handles incoming size update events for a specific resource.
+   * It adds the CID to the temporary persistent index (if not already processed
+   * or pending full recalc) and updates the in-memory delta map (`updatedSizeMap`).
+   *
+   * IMPORTANT: This should only be called for keys where this is relevant,
+   * such as `_private_size_` keys.
+   */
+  'worker/updateSizeSideEffects': async ({ resourceID, size }: { resourceID: string, size: number }) => {
+    // Ignore if the resource is already flagged for a full recalculation.
+    // Its size will be fully computed in `computeSizeTask`.
+    if (updatedSizeList.has(resourceID)) return
+
+    const current = updatedSizeMap.get(resourceID)
+    if (current === undefined) {
+      // First delta update for this resourceID since the last delta task ran.
+      // Add it to the persistent temporary index to ensure it's processed
+      // even if the worker restarts.
+      try {
+        await addToTempIndex(resourceID)
+        // Store the initial delta size.
+        updatedSizeMap.set(resourceID, size)
+      } catch (e) {
+        console.error(`Error adding ${resourceID} to temp index:`, e)
```

Comment:
Could you please prefix this string like so:

```js
console.error(e, `[ownerSizeTotalWorker] Error adding ${resourceID} to temp index`)
```

Also, please test to see if `console` here uses `pino` by forcing this error (or another error in a different worker) to throw that places the `e` at the start. If `pino` is used, then you should see a pretty-print output of it. If it's not being used, please update the PR so that it is.